### ISSUE TITLE
Do Top Hits disappear in pheno page?

### ISSUE THREAD

**Issue by phuwanat** [statgen/pheweb#46]
Hello,

It seems like there is no top hits shown in my pheno page. Is there any known issues about this?

Regards,
Phuwanat

**Comment by pjvandehaar** [comment]
If your pheweb is public, could you send me a link?

**Comment by phuwanat** [comment]
Thanks for very fast replying. I am still testing it on my machine. I don't have access to public server.

**Comment by pjvandehaar** [comment]
How did you install pheweb?

**Comment by phuwanat** [comment]
I use the latest one (0.9.13) by installing it from git with some modifications on significant threshold (https://github.com/phuwanat/pheweb/).

**Comment by phuwanat** [comment]
I tried to comment `_.filter(variants, _.property('show_gene'));` in function `populate_streamtable` of `pheno.js` and it showed all of variants. I am not sure whether it related or not but I haven't found point that insert `show_gene` in JSON data. Please forgive me if I did wrong.

Thanks,
Phuwanat

**Comment by pjvandehaar** [comment]
Thanks for that catch!  You're absolutely right, I removed `show_gene` about a week ago, but it didn't make any tests fail and I didn't reload the data on our main phewebs, so I didn't notice that I'd broken that table.

I was using `show_gene` so that I'd only show one variant per hit/loci in that table.  I removed while changing things in commit 029baa834c42e895351af17c1ab140c7f9ba6c4a.  I shouldn't have done that, because I liked that feature.

I'll add that feature back in later, but for now I've copied the fix that you have.
